static/templates/settings/admin_bot_form.hbs

```diff
@@ -1,19 +1,19 @@
 <div id="bot-edit-form" data-user-id="{{user_id}}">
     <form class="new-style edit_bot_form form-horizontal name-setting">
         <input type="hidden" name="is_full_name" value="true" />
-        <div class="input-group edit_bot_owner_container">
-            <label for="bot_owner_select">{{t "Owner" }}</label>
-            {{> dropdown_list_widget
-              widget_name="edit_bot_owner"
-              list_placeholder=(t 'Filter users')}}
-        </div>
-        <div class="name_change_container">
+        <div class="input-group name_change_container">
             <label for="full_name">{{t "Full name" }}</label>
             <input type="text" autocomplete="off" name="full_name" value="{{ full_name }}" />
         </div>
-        <div class="email_change_container">
+        <div class="input-group email_change_container">
```

Comment:
I think we can rename this class, since we cannot actually change the email.

Reply:
Renamed it to "email_container".
Thanks